javascript/general_style_guide.md

```diff
@@ -106,9 +118,9 @@ with its pros and cons. More info in the [React Docs](https://reactjs.org/docs/h
 
 ### Naming
 
-- Files: snake_case
+- Regular JS Files (not components): camelCase.js
+- Component Files: PascalCase
```

Comment:
how about the folder names? do we keep them underscore and lowercase?

Reply:
I don't have any specific opinion on that, but since @ubergesundheit  is going to work on it, let's agree on something... ¿What about pascal case for folders containing components and lowercase for folders not containing components?

I like this way because the folder name gives you information of what it's inside.

Reply:
I guess either is fine with me but using camelCase for folders would be more in line with the filename convention. I like using PascalCase for folders containnig components.

Reply:
I like it!

Reply:
ok, I can get used to that!

Reply:
oh and then the folder should always have a file in it with the same name, which will be the 'root' component right. I used to use `index.js` for that, but we want to avoid that right?


So if we have a folder: `ClusterView` then there should always be a file `ClusterView.js` in there too.

Reply:
That would be ideal IMO